### ISSUE TITLE
this may or may not be of use...

### ISSUE THREAD

**Issue by tkellen** [gruntjs/grunt-legacy-file#1]
...for replacing expandMapping.

https://github.com/tkellen/js-configfiles

If you want to use the code directly here--cool. If you want to consume it as a third party lib, cool. If you don't want to use it at all, that's fine too!

**Comment by jonschlinkert** [comment]
yeah I had that in mind, thanks. I'm a fan of using external libs whenever it makes sense. Do you remember off-hand if there are any feature/API differences?

**Comment by tkellen** [comment]
Backwards compatibility was my aim, but it hasn't been tested in the forge of the Grunt community yet (this is used in grunt-next). I'll add you as an author on the repo and npm.

**Comment by jonschlinkert** [comment]
k, I'm still working through unit tests, and will probably not be doing any major changes here in the near term, but will take a look at configfiles when we get to that point.

**Comment by tkellen** [comment]
:+1:
